app/javascript/react/components/molecules/StationValueLabel/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./StationValueLabel";
```

Comment:
@jankrzempek  and @smialko do you have your VSCode set up the way it's adding an empty line after last line of code? If not, could you set it up. We could also agree for some VSCode common config in a file. If you are willing to prepare it, please do so in a separate PR. :)

Reply:
Setting up right now